.changeset/tall-cows-ring.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Is this what we have to do now that we have a changeset rule on PRs?

Reply:
Yes, that's correct.

Reply:
Do we have a handy command to make this easier, like `yarn nochangeset`?